pythonforandroid/graph.py

```diff
@@ -133,13 +271,14 @@ def get_recipe_order_and_bootstrap(ctx, names, bs=None):
             raise
         orders.append(list(order))
 
-    # prefer python2 and SDL2 if available
+    # prefer python3 and SDL2 if available
     orders = sorted(orders,
-                    key=lambda order: -('python2' in order) - ('sdl2' in order))
+                    key=lambda order: -('python3' in order) - ('sdl2' in order))
```

Comment:
Nice! I've been waiting this one for a long time and never dared to touch it :smile: 
I remember I had to update the default in the conditional rebuild script however 92fde71d1621c4428b509731cf4d78ddb61b7acd

Reply:
No no you don't need to change the CI